### ISSUE TITLE
Warning Up window panel

### ISSUE THREAD

**Issue by SorbonKH** [k-box/k-box#156]
### Short Description of the issue
Usually some of users (deliberately/undeliberate) hook up the collection and sub collection during the work and will be tricky to find out who did it and resuming.
_write here what is going on_
it's going usually in the project colone 
### Expected behavior
when someone move the collections would be perfect if indicating "warning panel"
_write here what should actually happen_
  if the warning panel is working then would be more less safety and someone will not doing it on purpose
### Steps to reproduce the incorrect behavior

_1. List_

_2. all the steps_

_3. that have lead to the incorrect behavior_

### General information

Version of your K-Box: 

URL of the page where the problem has occured:

Type and version of your browser: 

In what language are you using your K-Box? 

### Screenshots

_add here any relevant screenshots, screencasts, etc_

**Comment by avvertix** [comment]
If I understand correctly you would like a message that asks to confirm the collection move operation. Is that true?

**Comment by shenriod** [comment]
Yes, correct, this is what we discussed with Sorbon! "Are you sure you want
to move the collection XY to the collection XZ"

**Comment by avvertix** [comment]
If I remember correctly initially we had such feature, but was removed because it was too intrusive when you have to re-organize entire collections

**Comment by shenriod** [comment]
I see. Then (as a new feature) I see 2 possible approaches:

1) Enforce the warning message only for "non admin" users (who anyway are very rarely moving collections around)
2) Enforce the warning for all users but allow the admin to lift up the warning for a limited period of time (only for himself or for all users). This way, the admin is also most of the time protected against non-voluntarily moving collections around but can actively "enter a danger zone" (kind of what "sudo" does on Linux systems), e.g. when he has to re-organize collections
